README.md

```diff
@@ -4,7 +4,7 @@
 
 Jamulus - Internet Jam Session Software
 =======================================
-<img align="left" src="src/res/homepage/mediawikisidebarlogo.png"/>
+<img align="left" src="src/res/homepage/mediasidebarlogo.png"/>
```

Comment:
The logo does not show up anymore:
![grafik](https://user-images.githubusercontent.com/46655886/97801038-24710680-1c3a-11eb-91b9-d3e07e63188f.png)

Reply:
Sorry. You have to change the file name back to mediawikisidebarlogo.png

Reply:
Done.